statline_bq/utils.py

```diff
@@ -6,12 +6,14 @@
 from datetime import datetime
 from shutil import rmtree
 from tempfile import gettempdir
+import xml.etree.ElementTree as ET
 
 import ndjson
 from dask.bag import Bag as DaskBag
 import dask.bag as db
 from pyarrow import json as pa_json
 import pyarrow.parquet as pq
+import pyarrow as pa
```

Comment:
not sure what best practice is, but I would be inclined just to
`import pyarrow as pa`

and call `pa.json`, `pa.parquet` etc. in the code

Reply:
I've tried that, but it doesn't work.

I get an `AttributeError`:  `module 'pyarrow' has no attribute 'json'`. I think this modules have to be explicitly imported to be used.